src/Source/Extras.swift

```diff
@@ -291,13 +291,17 @@ public extension String
     public func snakeCaseToCamelCase() -> String {
         let items = self.components(separatedBy: "_")
         var camelCase = ""
-        items.enumerated().forEach {
-            if ($1.isInteger()) {
+        for (i, x) in items.enumerated() {
+            if i == 0 {
+                camelCase += x
+                continue
```

Comment:
It may be possible to use a guard here
```guard i != 0 else {
    camelCase += x
    continue
}```
Just syntacticly swifty (asked Kevin about this)